erpnext/erpnext_integrations/taxjar_integration.py

```diff
@@ -260,3 +263,26 @@ def sanitize_error_response(response):
 		response = response.replace(k, v)
 
 	return response
+
+def create_taxjar_integration_request(doc, method):
+	if method == 'validate':
+		doc_name = ""
+	else:
+		doc_name = doc.name
+
+	if method == 'on_cancel':
+		status = "Cancelled"
+	else:
+		status = "Completed"
+
+	data = str(doc.as_dict())
```

Comment:
- JSON dumps ?

Reply:
Yes, To store all the doc related information for technical or tax-related issues.